src/PrettierEditProvider.ts

```diff
@@ -86,8 +86,11 @@ class PrettierEditProvider implements
     ): TextEdit[] {
         try {
             return [TextEdit.replace(
-                range,
-                format(document.getText(range), document.fileName)
+                fullDocumentRange(document),
+                format(document.getText(), document.fileName, {
+                  rangeStart: document.offsetAt(range.start),
+                  rangeEnd: document.offsetAt(range.end)
```

Comment:
Ahah. That was the range to 0 based function you told me in [your PR](https://github.com/prettier/prettier/pull/1637#issuecomment-302874022) 👍